src/components/AsideSection/AsideSection.tsx

```diff
@@ -25,15 +25,19 @@ export interface AsideSectionProps extends NestableBaseComponentProps {
  */
 export default class AsideSection extends React.Component<AsideSectionProps> {
   public render() {
+    const styles = mergeStyleSets(getStyles());
```

Comment:
According to Fabric's guidelines, this could be costly if re-rendered frequently. Should we be memoizing everywhere where we use `mergeStyleSets`? https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/merge-styles#optimizing-for-performance

Reply:
Yeah, it's probably a good idea. I've made the change.